site2/tools/build-site.sh

```diff
@@ -50,11 +50,8 @@ bash scripts/split-version-build.sh $@
 cd "$WEBSITE_DIR"
 
 CONTENT_DIR="$GEN_SITE_DIR"/content
-
-rm -rf "$CONTENT_DIR"
-mkdir -p "$CONTENT_DIR"
-mkdir -p "$CONTENT_DIR"/reference
-rsync -a ./docsify/ "$CONTENT_DIR"/reference
+rm -rf "$CONTENT_DIR" && mkdir -p "$CONTENT_DIR"
+cp -R static/reference "$CONTENT_DIR"
 rsync -a ./build/ "$CONTENT_DIR"
```

Comment:
`cp -R` seems work differently to the standard on the CI env. Use `rsync -a` correctly stably copy all files (including hidden files) recursively.

Besides, since we move references under the `static` folder, there's no need to special handle them.

Thus, this code snippet can be written as:

```bash
rm -rf "$CONTENT_DIR"
mkdir -p "$CONTENT_DIR"
rsync -a ./build/ "$CONTENT_DIR"
```

@urfreespace @SignorMercurio please check.

Reply:
yes, thanks for @tisonkun, could you pls do a test with this code from @tisonkun's suggestion? @SignorMercurio

Reply:
Sure, I'll have a try and update the script.